2-mandatory/5-eligible-students.js

```diff
@@ -6,8 +6,35 @@
     (see tests to confirm how this data will be structured)
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
+function onlyNames(nameClassesPair) {
+  return nameClassesPair[0];
+}
+
+
+function isEligible(student) {
+  return student[1]>7;
+};
+
+function getEligibleStudents(studentAttendance) {
+  let sitInExam = studentAttendance.filter(isEligible);
+  let examNames = sitInExam.map(onlyNames);
+  return examNames;
+};
```

Comment:
Same issue here. The logic works absolutly fine. We don't have to have the fisrt two functions. Fewer lines are not always better. But, fewer lines are often better than more lines in terms of readability and maintainability.